frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -3,28 +3,30 @@ import * as _ from 'lodash';
 import { pluralize } from '@patternfly/react-core';
 import { ChartDonut } from '@patternfly/react-charts';
 import { SecurityIcon } from '@patternfly/react-icons';
-import { URLHealthHandler } from '@console/plugin-sdk';
+import { ResourceHealthHandler } from '@console/plugin-sdk';
 import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
-import { FirehoseResult } from '@console/internal/components/utils/types';
 import { Link } from 'react-router-dom';
-import { referenceForModel } from '@console/internal/module/k8s';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
 import { ImageManifestVuln } from '../types';
 import { vulnPriority } from '../const';
 import { ImageManifestVulnModel } from '../models';
+import {
+  ResourcesObject,
+  WatchK8sResults,
+} from '@console/internal/components/utils/k8s-watch-hook';
 
-export const securityHealthHandler: URLHealthHandler<string> = (
-  k8sHealth,
-  error,
-  resource: FirehoseResult<ImageManifestVuln[]>,
-) => {
-  if (error || _.get(resource, 'loadError')) {
+export const securityHealthHandler: ResourceHealthHandler<ResourcesObject> = (resourceObject) => {
```

Comment:
same here, you should be more specific with the type. Since we will use it it multiple places, you should add the type to `types.ts` https://github.com/openshift/console/blob/master/frontend/packages/container-security/src/types.ts

something like 
```
export type WatchImageVuln = {
  imageManifestVuln: ImageManifestVuln[];
};
```

then

```suggestion
export const securityHealthHandler: ResourceHealthHandler<WatchImageVuln> = (resourceObject) => {
```